examples/plugin-for-example/README.md

```diff
@@ -0,0 +1,33 @@
+# WASI plugin example
+
+In this example we extend the imports of Wasmer's WASI ABI to demonstrate how custom plugins work.
+
+See the `wasmer/examples/plugin.rs` file for the source code of the host system.
+
+## Compiling
+
+```
+# Install an up to date version of Rust nightly
+# Add the target
+rustup target add wasm32-unknown-wasi
+# build it
+cargo build --release --target=wasm32-unknown-wasi
+# copy it to examples folder
+cp ../../target/wasm32-unknown-wasi/release/plugin-for-example.wasm ../
```

Comment:
It may also be worth noting that WASI is only supported on windows with the gnu toolchain. If you can't find a way to work this detail into the example, that's ok, but it could be another stumbling block for some.

Reply:
Okay, I'm not sure exactly what to add to say that.  What I mean is, does that show up in the `target` dir?